backend/xivdpscalc/pps/sample_sch_rotation.py

```diff
@@ -67,8 +67,10 @@ def get_action(self, current_time: ElapsedTime, cooldowns: dict[SchAction, float
                 if cooldowns[action] <= earliest_nonclip_cast_time and constraint:
                     selected_action = action
                     break
-        # if bio needs refreshing
-        elif remaining_effect_durations[SchEffect.BIOLYSIS] <= 0:
+        # if bio fell off
```

Comment:
this needs a bit better documentation for why it's this way:
refusing to reapply bio when aetherflow is between 1-2 gcds on cd allows odd-numbered gcd-per-min rotations to correctly delay bio reapplication and keep it synced with aetherflow